pkg/cgroups/pids.go

```diff
@@ -46,10 +46,13 @@ func (c *pidHandler) Destroy(ctr *CgroupControl) error {
 func (c *pidHandler) Stat(ctr *CgroupControl, m *Metrics) error {
 	var PIDRoot string
 
-	if ctr.cgroup2 {
+	if ctr.cgroup2 { // nolint: gocritic
 		PIDRoot = filepath.Join(cgroupRoot, ctr.path)
-	} else {
+	} else if ctr.path != "" {
```

Comment:
we could skip it also on cgroup v2.  Since there is no `pids.current` in the cgroup v2 root

Reply:
Alright, changed it as suggested.